src/main/java/com/hrznstudio/galacticraft/api/internal/mixin/ClientParticleMixin.java

```diff
@@ -0,0 +1,31 @@
+package com.hrznstudio.galacticraft.api.internal.mixin;
+
+import com.hrznstudio.galacticraft.api.celestialbodies.CelestialBodyType;
+import net.minecraft.client.particle.Particle;
+import net.minecraft.client.world.ClientWorld;
+import net.minecraft.util.registry.RegistryKey;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import java.util.Optional;
+
+@Mixin(Particle.class)
+public abstract class ClientParticleMixin {
+
+    @Shadow
+    protected float gravityStrength;
+
+    @Inject(method = "<init>", at = @At("RETURN"))
```

Comment:
You need to specify the parameters as there is more than one constructor.
```suggestion
    @Inject(method = "<init>(Lnet/minecraft/client/world/ClientWorld;DDD)V", at = @At("RETURN"))
```